src/components/AddLayerModal/index.tsx

```diff
@@ -41,6 +46,8 @@ import {
 } from '../../store/addLayerModal';
 
 import './index.less';
```

Comment:
Please organize and probably also sort imports.

Reply:
The imports are now organized!